sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -0,0 +1,266 @@
+{
+  "/@default": "entry",
+  "/ENTRY[entry]/@default": "data",
+  "/ENTRY[entry]/definition": "NXmpes",
+  "/ENTRY[entry]/definition/@version": "None",
+  "/ENTRY[entry]/title": "@attrs:metadata/scientificMetadata/title",
+  "/ENTRY[entry]/start_time": "@attrs:metadata/creationTime",
+  "/ENTRY[entry]/experiment_institution": "Deutsches Elektronen-Synchrotron DESY",
+  "/ENTRY[entry]/experiment_facility": "Free-Electron Laser FLASH",
+  "/ENTRY[entry]/experiment_laboratory": "@attrs:metadata/creationLocation",
+  "/ENTRY[entry]/entry_identifier": "@attrs:metadata/_id",
+  "/ENTRY[entry]/USER[user0]": {
+    "name": "@attrs:metadata/pi/last_name",
+    "role": "Principal investigator",
+    "affiliation": "@attrs:metadata/pi/institute",
+    "email": "@attrs:metadata/pi/email"
+  },
+  "/ENTRY[entry]/USER[user1]": {
+    "name": "@attrs:metadata/leader/last_name",
```

Comment:
@kutnyakhov i changed from lastname to last_name here. Its because spelling check complains otherwise. You might need to adapt the tutorial

Reply:
Anyways did not exist.